.nix/coq-overlays/elpi/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchzip, buildDunePackage, camlp5
```

Comment:
This should be in `.nix/ocaml-overlays`

Reply:
why not just `.nix/overlays/` ?

Reply:
because I have to know where to put it in the nixpkgs archive

Reply:
`.nix/overlay` would put it outside of `coqPackages` and `ocamlPackages`, and that would be problematic since elpi already exists inside `ocamlPackages`